src/Management/src/Endpoint/Actuators/HttpExchanges/Diagnostics/DiagnosticObserver.cs

```diff
@@ -3,27 +3,26 @@
 // See the LICENSE file in the project root for more information.
 
 using System.Diagnostics;
-using System.Reflection;
 using Microsoft.Extensions.Logging;
 
-namespace Steeltoe.Management.Endpoint.Actuators.Metrics.Diagnostics;
+namespace Steeltoe.Management.Endpoint.Actuators.HttpExchanges.Diagnostics;
 
-public abstract class DiagnosticObserver : IDiagnosticObserver
+internal abstract class DiagnosticObserver : IObserver<KeyValuePair<string, object?>>, IDisposable
 {
     private readonly ILogger _logger;
-    private IDisposable? _subscription;
 
-    public string ListenerName { get; }
-    public string ObserverName { get; }
+    private readonly string _observerName;
```

Comment:
This is what it looks like now:

![image](https://github.com/user-attachments/assets/6c7c5813-5c27-491e-ae61-759e9e5dd478)

Why the blank line after `_logger`, and why not follow the same order as the ctor parameters?